hydrofunctions/flood_stage.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+ResponseFormat = "json"  # json, xml
+
+# OLD web service dating to 2016; discontinued for new one?
+url = "https://waterwatch.usgs.gov/webservices/floodstage"
+
+
+def get_flood_stages(res_fmt: str = ResponseFormat):
```

Comment:
```suggestion
def get_flood_stages(format: str = "json"):
```
By putting the default option right in the call signature, it is easier for the user to see the option.

Reply:
True